src/test/pmem_memcpy/out0.log.match

```diff
@@ -0,0 +1,3 @@
+pmem_memcpy/TEST0: START: pmem_memcpy
+ ./pmem_memcpy$(*) $(*)/testfile1 0 0 4096
```

Comment:
The second line of this match file should look like this:
 ./pmem_memcpy$(nW) $(nW)/testfile1 0 0 4096

 so a token for ".static-(non)debug" should be $(nW) and a token for "$DIR" should be $(nW) too.
 
 There are more occurrences of the same issue:
./src/test/pmem_memcpy/out1.log.match:2: ./pmem_memcpy$(*) $(*)/testfile1 7 0 4096
./src/test/pmem_memcpy/out2.log.match:2: ./pmem_memcpy$(*) $(*)/testfile1 7 9 4096
./src/test/pmem_memcpy/out3.log.match:2: ./pmem_memcpy$(*) $(*)/testfile1 0 9 4096
./src/test/pmem_memmove/out0.log.match:2: ./pmem_memmove$(*) $(*)/testfile1 b:4096
./src/test/pmem_memmove/out1.log.match:2: ./pmem_memmove$(*) $(*)/testfile1 s:7 b:4096
./src/test/pmem_memmove/out2.log.match:2: ./pmem_memmove$(*) $(*)/testfile1 d:7 s:13 b:4096
./src/test/pmem_memmove/out3.log.match:2: ./pmem_memmove$(*) $(*)/testfile1 b:4096 S:23 o:1
./src/test/pmem_memmove/out4.log.match:2: ./pmem_memmove$(*) $(*)/testfile1 b:4096 d:21
./src/test/pmem_memmove/out5.log.match:2: ./pmem_memmove$(*) $(*)/testfile1 b:4096 d:21 s:7
./src/test/pmem_memmove/out6.log.match:2: ./pmem_memmove$(*) $(*)/testfile1 b:4096 o:1 S:20
./src/test/pmem_memmove/out7.log.match:2: ./pmem_memmove$(*) $(*)/testfile1 b:4096 d:13 o:1 S:20
./src/test/pmem_memmove/out8.log.match:2: ./pmem_memmove$(*) $(*)/testfile1 b:2048 d:13 o:2 S:20
./src/test/pmem_memmove/out9.log.match:2: ./pmem_memmove$(*) $(*)/testfile1 b:4096 o:2 S:20
./src/test/pmem_memset/out0.log.match:2: ./pmem_memset$(*) $(*)/testfile1 0 4096
./src/test/pmem_memset/out1.log.match:2: ./pmem_memset$(*) $(*)/testfile1 13 4096

This change was introduced by the following two commits:

commit e0db9beda0d0671fff52b9644084324c99e4f8db
test: change testname$(*) to testname$(nW) in match files

commit 4662353cc0a0c3cce5a7b7d6a8bdb97e3237ffbb
test: change token for $DIR in match files